### PR TITLE
My code review suggestions.

### PR DESCRIPTION
**Pull request by petromw**
Description of the project: 
From my understanding, this project is a way to take attendance in class and share notes.

This project was available and compiled. There was not a lot of documentation, but it was fairly easy to understand.

During the code review, I learned about:
* How to activate the camera
* The different visibility modifiers in kotlin
* The order of things within a class

My changes:
* I changed the visibility modifiers to align with best practices.
* I moved the companion object to the bottom of the class to align with best practices.

### CODE REVIEW

File: app/src/main/java/edu/uc/clemenks/keepingupwithschool/ui/main/MainFragment.kt [hunk 1]
```diff
@@ -93,4 +89,7 @@ class MainFragment : Fragment() {
         }
     }
 
+    companion object {
+        fun newInstance() = MainFragment()
+    }
```
Comment (petromw) [inline app/src/main/java/edu/uc/clemenks/keepingupwithschool/ui/main/MainFragment.kt:94]: I moved the companion object based on the suggestion here: https://kotlinlang.org/docs/reference/coding-conventions.html

File: app/src/main/java/edu/uc/clemenks/keepingupwithschool/ui/main/MainFragment.kt [hunk 2]
```diff
@@ -20,12 +20,8 @@ import kotlinx.android.synthetic.main.main_fragment.*
...
 class MainFragment : Fragment() {
 
-    val CAMERA_PERMISSION_REQUEST_CODE = 1999
-    val CAMERA_REQUEST_CODE = 1998
-
-    companion object {
-        fun newInstance() = MainFragment()
-    }
+    private val CAMERA_PERMISSION_REQUEST_CODE = 1999
+    private val CAMERA_REQUEST_CODE = 1998
```
Comment (petromw) [inline app/src/main/java/edu/uc/clemenks/keepingupwithschool/ui/main/MainFragment.kt:24]: I changed the visibilty modifiers based on the suggestion here: https://kotlinlang.org/docs/reference/visibility-modifiers.html

Comment (discospiff) [reply app/src/main/java/edu/uc/clemenks/keepingupwithschool/ui/main/MainFragment.kt:24]: Good idea... and nice citation.

**Comment by discospiff** [comment]
A couple of good suggestions, worth merging.

**Comment by Clemenks** [comment]
Added Manually

**Pull request closed without merging**